features/supplier/view_and_edit_services.feature

```diff
@@ -0,0 +1,27 @@
+@view_dos_service
+Feature: Supplier being able to view their DOS services
+
+Background:
+  Given I have a live digital outcomes and specialists framework
+  And I have a supplier
+  And that supplier is logged in
+  Given that supplier is on that framework
+
+Scenario Outline: Supplier coming from dashboard to view the detail page for their <lot_slug> service
+  Given that supplier has a service on the <lot_slug> lot
+  And I am on the /suppliers page
+  When I click 'View'
+  Then I am on the 'Current services' page
+  When I click '<service_name>'
+
```

Comment:
I had a blank line for legibility as I saw it used this way in requirements.feature, but I deleted it now.